zone/bot.cpp

```diff
@@ -4064,6 +4063,27 @@ void Bot::GetBotItems(EQ::InventoryProfile &inv, std::string* error_message)
 	UpdateEquipmentLight();
 }
 
+std::map<uint16, uint32> Bot::GetBotItemSlots()
+{
+	std::map<uint16, uint32> m;
+	if (!GetBotID()) {
+		return m;
+	}
+
+	if (!database.botdb.LoadItemSlots(GetBotID(), m)) {
+		GetBotOwner()->CastToClient()->Message(
+			Chat::White,
+			fmt::format(
+				"Failed to load inventory slots for {}.",
+				GetCleanName()
+			).c_str()
+		);
+		return m;
```

Comment:
This return seems redundant because the fallback will end up returning the same thing

Reply:
Makes sense to me.